packages/blade/src/components/PopupArrow/PopupArrow.web.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { FloatingArrow } from '@floating-ui/react';
+import type { PopupArrowProps } from './types';
+import { useTheme } from '~utils';
+
+const PopupArrow = React.forwardRef<SVGSVGElement, PopupArrowProps>(
```

Comment:
I think this is fine, Floating I had to refactor this out because of Native implementation is custom since FloatingUI doesn't provide that, so it's not exactly a "Floating" popup. 

Also it's closer to the Popover in the editor this way: 
<img width="305" alt="image" src="https://github.com/razorpay/blade/assets/35374649/acf8529d-ef3c-4a65-a042-fba9c3581cee">